htmd/parameterization/dihedral.py

```diff
@@ -532,35 +455,11 @@ def test_getValidQMResults(self):
         self.assertEqual(len(self.df._getValidQMResults()[0]), 17)
 
     def test_getBounds(self):
-
         self.df.dihedrals = [[0, 1, 2, 3]]
         lower_bounds, upper_bounds = self.df._getBounds()
         self.assertListEqual(list(lower_bounds), [0.,  0.,  0.,  0.,  0.,  0.,   0.,   0.,   0.,   0.,   0.,   0., -10.])
         self.assertListEqual(list(upper_bounds), [10., 10., 10., 10., 10., 10., 360., 360., 360., 360., 360., 360.,  10.])
 
-    def test_paramsToVector(self):
```

Comment:
The methods are not removed, so the tests have to be preserved.

Reply:
Readded and updated for the new code